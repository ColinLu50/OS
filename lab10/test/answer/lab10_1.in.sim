15119
437
437
366
437
365
