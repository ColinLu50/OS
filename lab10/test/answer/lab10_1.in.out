FCFS
65 -> 135 -> 29 -> 344 -> 73 -> 222 -> 4 -> 200 -> 11 -> 236 -> 228 -> 171 -> 16 -> 298 -> 90 -> 242 -> 120 -> 4 -> 184 -> 46 -> 227 -> 119 -> 369 -> 345 -> 52 -> 229 -> 372 -> 56 -> 253 -> 315 -> 99 -> 141 -> 213 -> 140 -> 246 -> 271 -> 364 -> 16 -> 316 -> 172 -> 163 -> 231 -> 323 -> 59 -> 176 -> 366 -> 108 -> 94 -> 323 -> 249 -> 318 -> 208 -> 72 -> 188 -> 0 -> 121 -> 135 -> 237 -> 126 -> 41 -> 170 -> 372 -> 140 -> 307 -> 316 -> 226 -> 249 -> 318 -> 60 -> 245 -> 4 -> 310 -> 176 -> 14 -> 239 -> 323 -> 152 -> 349 -> 246 -> 163 -> 262 -> 170 -> 302 -> 184 -> 73 -> 34 -> 39 -> 89 -> 248 -> 129 -> 266 -> 327 -> 115 -> 352 -> 141 -> 46 -> 265 -> 260 -> 293 -> 49 -> 31 -> 184 -> 121 -> 9 -> 58 -> 328 -> 210 -> 16
15119
SSTF
65 -> 60 -> 59 -> 58 -> 56 -> 52 -> 49 -> 46 -> 46 -> 41 -> 39 -> 34 -> 31 -> 29 -> 16 -> 16 -> 16 -> 14 -> 11 -> 9 -> 4 -> 4 -> 4 -> 0 -> 72 -> 73 -> 73 -> 89 -> 90 -> 94 -> 99 -> 108 -> 115 -> 119 -> 120 -> 121 -> 121 -> 126 -> 129 -> 135 -> 135 -> 140 -> 140 -> 141 -> 141 -> 152 -> 163 -> 163 -> 170 -> 170 -> 171 -> 172 -> 176 -> 176 -> 184 -> 184 -> 184 -> 188 -> 200 -> 208 -> 210 -> 213 -> 222 -> 226 -> 227 -> 228 -> 229 -> 231 -> 236 -> 237 -> 239 -> 242 -> 245 -> 246 -> 246 -> 248 -> 249 -> 249 -> 253 -> 260 -> 262 -> 265 -> 266 -> 271 -> 293 -> 298 -> 302 -> 307 -> 310 -> 315 -> 316 -> 316 -> 318 -> 318 -> 323 -> 323 -> 323 -> 327 -> 328 -> 344 -> 345 -> 349 -> 352 -> 364 -> 366 -> 369 -> 372 -> 372
437
SCAN
65 -> 60 -> 59 -> 58 -> 56 -> 52 -> 49 -> 46 -> 41 -> 39 -> 34 -> 31 -> 29 -> 16 -> 14 -> 11 -> 9 -> 4 -> 0 -> 0 -> 72 -> 73 -> 89 -> 90 -> 94 -> 99 -> 108 -> 115 -> 119 -> 120 -> 121 -> 126 -> 129 -> 135 -> 140 -> 141 -> 152 -> 163 -> 170 -> 171 -> 172 -> 176 -> 184 -> 188 -> 200 -> 208 -> 210 -> 213 -> 222 -> 226 -> 227 -> 228 -> 229 -> 231 -> 236 -> 237 -> 239 -> 242 -> 245 -> 246 -> 248 -> 249 -> 253 -> 260 -> 262 -> 265 -> 266 -> 271 -> 293 -> 298 -> 302 -> 307 -> 310 -> 315 -> 316 -> 318 -> 323 -> 327 -> 328 -> 344 -> 345 -> 349 -> 352 -> 364 -> 366 -> 369 -> 372
437
C-SCAN
65 -> 60 -> 59 -> 58 -> 56 -> 52 -> 49 -> 46 -> 41 -> 39 -> 34 -> 31 -> 29 -> 16 -> 14 -> 11 -> 9 -> 4 -> 0 -> 0 -> 373 -> 372 -> 369 -> 366 -> 364 -> 352 -> 349 -> 345 -> 344 -> 328 -> 327 -> 323 -> 318 -> 316 -> 315 -> 310 -> 307 -> 302 -> 298 -> 293 -> 271 -> 266 -> 265 -> 262 -> 260 -> 253 -> 249 -> 248 -> 246 -> 245 -> 242 -> 239 -> 237 -> 236 -> 231 -> 229 -> 228 -> 227 -> 226 -> 222 -> 213 -> 210 -> 208 -> 200 -> 188 -> 184 -> 176 -> 172 -> 171 -> 170 -> 163 -> 152 -> 141 -> 140 -> 135 -> 129 -> 126 -> 121 -> 120 -> 119 -> 115 -> 108 -> 99 -> 94 -> 90 -> 89 -> 73 -> 72
366
LOOK
65 -> 60 -> 59 -> 58 -> 56 -> 52 -> 49 -> 46 -> 41 -> 39 -> 34 -> 31 -> 29 -> 16 -> 14 -> 11 -> 9 -> 4 -> 0 -> 72 -> 73 -> 89 -> 90 -> 94 -> 99 -> 108 -> 115 -> 119 -> 120 -> 121 -> 126 -> 129 -> 135 -> 140 -> 141 -> 152 -> 163 -> 170 -> 171 -> 172 -> 176 -> 184 -> 188 -> 200 -> 208 -> 210 -> 213 -> 222 -> 226 -> 227 -> 228 -> 229 -> 231 -> 236 -> 237 -> 239 -> 242 -> 245 -> 246 -> 248 -> 249 -> 253 -> 260 -> 262 -> 265 -> 266 -> 271 -> 293 -> 298 -> 302 -> 307 -> 310 -> 315 -> 316 -> 318 -> 323 -> 327 -> 328 -> 344 -> 345 -> 349 -> 352 -> 364 -> 366 -> 369 -> 372
437
C-LOOK
65 -> 60 -> 59 -> 58 -> 56 -> 52 -> 49 -> 46 -> 41 -> 39 -> 34 -> 31 -> 29 -> 16 -> 14 -> 11 -> 9 -> 4 -> 0 -> 372 -> 369 -> 366 -> 364 -> 352 -> 349 -> 345 -> 344 -> 328 -> 327 -> 323 -> 318 -> 316 -> 315 -> 310 -> 307 -> 302 -> 298 -> 293 -> 271 -> 266 -> 265 -> 262 -> 260 -> 253 -> 249 -> 248 -> 246 -> 245 -> 242 -> 239 -> 237 -> 236 -> 231 -> 229 -> 228 -> 227 -> 226 -> 222 -> 213 -> 210 -> 208 -> 200 -> 188 -> 184 -> 176 -> 172 -> 171 -> 170 -> 163 -> 152 -> 141 -> 140 -> 135 -> 129 -> 126 -> 121 -> 120 -> 119 -> 115 -> 108 -> 99 -> 94 -> 90 -> 89 -> 73 -> 72
365
